### PR TITLE
Clean up guest state bsc#1221917

### PR DESCRIPTION
**Pull request by tbaev**
- Related ticket: https://progress.opensuse.org/issues/164189
- Needles: N/A
- Verification run: [15-SP5-XEN](http://openqa.qam.suse.cz/tests/78456) , [15-SP6-XEN](http://openqa.qam.suse.cz/tests/78447)

### CODE REVIEW

**Comment by RoyCai7** [comment]
LGTM

File: tests/virtualization/universal/virtmanager_rm_devices.pm [hunk 1]
```diff
@@ -62,12 +63,18 @@ sub run_test {
...
+
+    # Workaround to return guests to initial sate, related to bsc#1221917
+    shutdown_guests();
+    foreach my $guest (keys %virt_autotest::common::guests) {
+        if ($guest =~ m/PV/i) {
+            record_soft_failure 'bsc#1221917 - [MU]Core Dump Occurs on Bare Metal SLES15 SP4/SP5/SP6 with Xen Following Disk Detachment in PV Guest Environment.';
+            assert_script_run "virsh detach-disk $guest xvdb --config";
+        }
+    }
+    start_guests();
```
Comment (alice-suse) [inline tests/virtualization/universal/virtmanager_rm_devices.pm:77]: Thank you for the fix, @tbaev .

I am a little bit confused now with the overall test in this virtmanager_rm_devices module. Maybe it is not caused by your PR... The bug is only effective for PV guest, right? Why do we stop test on both PV and HVM?
If this is indeed a problem but not by my misunderstanding, can you fix it together?

Comment (alice-suse) [reply tests/virtualization/universal/virtmanager_rm_devices.pm:77]: I think I know why now. The original test only test PV guest. So after excluding it, nothing is tested.

**Review by alice-suse — APPROVED** [review]
LGTM

File: tests/virtualization/universal/virtmanager_rm_devices.pm (hunk 1 above)
Comment (tbaev) [reply tests/virtualization/universal/virtmanager_rm_devices.pm:77]: HVM testing in this module was excluded long time ago in this PR ( 2019 year ), not sure why there is no ticket. Maybe we can crate a ticket to research it a little bit more.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8783

Comment (alice-suse) [reply tests/virtualization/universal/virtmanager_rm_devices.pm:77]: Thank you @tbaev . Please give the new ticket normal priority .

**Pull request merged**